apps/docs-snippets2/src/cookbook/resubmitting-failed-transactions/wrong-resubmission.ts

```diff
@@ -0,0 +1,36 @@
+import type { FuelError } from 'fuels';
+import { bn, Provider, Wallet } from 'fuels';
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from 'src/env';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+
+const baseAssetId = provider.getBaseAssetId();
+const transferAmount = 1000;
+
+// #region resubmitting-failed-transactions-3
+const transactionRequest = await wallet.createTransfer(
```

Comment:
This is also not pastable, but could be included in a _full example_

Reply:
@maschad I don’t think this page needs the full snippet. After adding it, the content feels overcrowded, making the snippet too lengthy. In this case, separate snippets seem to work better.

What do you think?

Reply:
I don't have a strong opinion on it, so that's fine with me.